providers/feature_dism.rb

```diff
@@ -50,13 +50,17 @@ def delete_feature(_name)
 
 def installed?
   @installed ||= begin
+    install_ohai_plugin unless node['dism_features']
```

Comment:
The ohai_plugin provider does a reload on the node object so it should be there after this